### PR TITLE
update meta description

### PR DESCRIPTION
**Pull request by metinbaris**
Closes ___ (e.g. #230)

## What does this PR do?

It changes the word "foodsaving" to "independent" in the meta description

## Links to related issues
https://github.com/yunity/karrot-frontend/issues/2375

## Checklist

- static values testing is kind of not needed
- shared pr with group chat

### CODE REVIEW

**Review by tiltec — APPROVED** [review]

File: src-cordova/config/dev/config.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' encoding='utf-8'?>
 <widget id="world.karrot.dev" version="9.3.0" xmlns="http://www.w3.org/ns/widgets" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:cdv="http://cordova.apache.org/ns/1.0">
     <name>Karrot Dev</name>
-    <description>Modern website for organization of foodsaving groups worldwide</description>
+    <description>Independent group organization</description>
     <author email="karrot@foodsaving.world" href="https://foodsaving.world">
```
Comment (brnsolikyl) [inline src-cordova/config/dev/config.xml:5]: I think info@karrot.world is more suitable. But I would not know where to put a link to the authorship actually, maybe point towards https://community.foodsaving.world/people

File: src-cordova/config/dev/config.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
         Foodsaving World Team
```
Comment (brnsolikyl) [inline src-cordova/config/dev/config.xml:6]: And change this for Karrot Team

File: src-cordova/config/prod/config.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' encoding='utf-8'?>
 <widget id="world.karrot" version="9.3.0" xmlns="http://www.w3.org/ns/widgets" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:cdv="http://cordova.apache.org/ns/1.0">
     <name>Karrot</name>
-    <description>Modern website for organization of foodsaving groups worldwide</description>
+    <description>Independent group organization</description>
     <author email="karrot@foodsaving.world" href="https://foodsaving.world">
```
Comment (brnsolikyl) [inline src-cordova/config/prod/config.xml:5]: Same as the above on the dev file

**Comment by nicksellen** [comment]
Hey! Thanks for getting started on this @metinbaris - we discussed and came up with some ideas for what it can say over at https://github.com/yunity/karrot-frontend/issues/2375#issuecomment-864930273 :)

**Comment by brnsolikyl** [comment]
Hello, @metinbaris ! I wonder if you gave up on this issue, and if so, why, if you don't mind me asking... I hope it didn't feel like our input was unclear or maybe too much.

**Pull request closed without merging**